mews-api.md

```diff
@@ -641,6 +641,10 @@ There are certain rules that need to be followed in order for Mews to process th
         {
           "code": "1",
           "cost": 20,
```

Comment:
Shouldn't we remove all the obsolete things from the examples and also from the documentation?

Reply:
@siroky I left them there until we remove them completely we have issue with deadline to remove them completely

Reply:
I know, usually in connector, we delete from documentation immediately, but support in code until deadline. Discuss with @vitasamek